packages/parser/src/schemas/transfer-family.ts

```diff
@@ -0,0 +1,25 @@
+import { z } from 'zod';
+
+/**
+ * TransferFamilySchema validates events coming from AWS Transfer Family.
```

Comment:
Please add an example of the event in the schema description like in all the other schemas, [example](https://github.com/aws-powertools/powertools-lambda-typescript/blob/66364b5f9c74df7eefc695aab788c1f4008e7250/packages/parser/src/schemas/cloudwatch.ts#L34-L79).